js/app.js

```diff
@@ -178,36 +178,31 @@ function handleFileButton(evt){
 
 Event.on(document.body, 'ui:click', '.open-files', handleFileButton);
 
+function handleExample(text, filename){
```

Comment:
This is a refactoring to make it easier to add examples without code duplication.